internal/manager/featuregates/feature_gates_test.go

```diff
@@ -23,6 +24,12 @@ func TestFeatureGates(t *testing.T) {
 	assert.NoError(t, err)
 	assert.True(t, fgs[GatewayAlphaFeature])
 
+	t.Log("Verifying feature gates setup will return error when settings has conflicts")
+	featureGates = map[string]bool{KongCustomEntity: true, FillIDsFeature: false}
+	_, err = New(setupLog, featureGates)
+	assert.Error(t, err)
```

Comment:
```suggestion
	require.Error(t, err)
```

Otherwise if `err` is nil, the line below will panic.

Reply:
done, and changed another appearance.